### PR TITLE
[Feat] Loading skeleton

### PR DESCRIPTION
**Pull request by antonyayoub**
# Name of the feature/issue
Loading skeleton after network change
## Description
Change of loading spinner to skeleton for Activity List, Assets List & Wallet Balance in the popup page

### CODE REVIEW

File: packages/ui/src/components/token/TokenSummary.tsx [hunk 1]
```diff
@@ -28,8 +30,38 @@ const TokenSummary: FC<{
 }
 
 const Balances = ({ children }: { children: React.ReactNode }) => {
+    const state = useBlankState()!
+
+    const isLoading =
+        state.isNetworkChanging || state.isRatesChangingAfterNetworkChange
+
+    const balancesLoadingSkeleton = (
```
Comment (fpezzola) [inline packages/ui/src/components/token/TokenSummary.tsx:38]: Can you export this to a new component?

Comment (fpezzola) [reply packages/ui/src/components/token/TokenSummary.tsx:38]: For me is easier to read this component in this way:


```javascript

return (
<>
{ isLoading ? <BalanceLoadingSkeleton />  : < div>{children} </div> }
</>
)

```

**Review by fpezzola — CHANGES_REQUESTED** [review]
Excellent work. Please check my comments, they should be easy to address

File: packages/ui/src/components/token/TokenSummary.tsx (hunk 1 above)
Comment (antonyayoub) [reply packages/ui/src/components/token/TokenSummary.tsx:38]: Make sense cleaner that way!

**Review by fpezzola — APPROVED** [review]
lgtm!

**Pull request merged**